### PR TITLE
Add 5.3 release notes

### LINKED ISSUE #1260: Sensu Go 5.3 release notes
Page: https://docs.sensu.io/sensu-go/5.3/release-notes/

To do: Add release notes

Reference:

- https://github.com/sensu/sensu-go/tree/master/CHANGELOGS
- https://github.com/sensu/sensu-go/blob/master/CHANGELOG.md
- https://github.com/sensu/sensu-enterprise-go/blob/master/CHANGELOG.md
- https://github.com/sensu/sensu-go/pull/2771

Synopsis:

The latest release of Sensu Go, version 5.3.0, is now available for download. This release has some very exciting feature additions and key bug fixes. 5.3.0 enables Active Directory to be configured as an authentication provider with a valid license key. Additionally, round robin scheduling has been fully re-implemented and is available for use.

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Adds release notes for Sensu Go 5.3

## Motivation and Context

Closes #1260

### CODE REVIEW

**Review by rgeniesse — APPROVED** [review]
LGTM. Links all work 💯

File: content/sensu-go/5.3/release-notes.md [hunk 1]
```diff
@@ -23,6 +24,43 @@ Read the [upgrading guide][1] for information on upgrading to the latest version
...
+
+- Round robin check scheduling allows Sensu to execute checks for each entity within a subscription in turn, giving you the ability to rotate a check through a set of entities. To enable round-robin scheduling, set the `round_robin` check attribute to `true`. See the [check reference][18] for more information.
+- Sensu now provides [enterprise-only][19] support for using Microsoft Active Directory as an external authentication provider. Read the [authentication guide][20] to configure Active Directory, and check out the [getting started guide][19] for more information about licensing.
+- The dashboard now features offline state detection and displays an alert banner in the event that the dashboard loses connection to the backend.
+
+**IMPROVEMENTS:**
+
+- The agent socket event format now supports the `handlers` attribute, giving you the ability to send socket events to a Sensu pipeline. See the [agent reference][21] to learn more about creating and handling monitoring events using the agent socket.
+- Assets now feature improved download performance using buffered I/O.
+- sensuctl now uses a 15-second timeout period when connecting to the Sensu backend.
```
Comment (nikkictl) [inline content/sensu-go/5.3/release-notes.md]: `sensuctl` maybe? Idk you would know better than me 😆

**Review by nikkictl — APPROVED** [review]
So good!

giphy-4

File: content/sensu-go/5.3/release-notes.md (hunk 1 above)
Comment (apaskulin) [inline content/sensu-go/5.3/release-notes.md]:
```suggestion
- The sensuctl CLI now uses a 15-second timeout period when connecting to the Sensu backend.
```

**Pull request merged**